flashinfer/moe_ep/kernel_src/sm120/split_cutedsl_megakernel/shim/runtime.py

```
@@ -0,0 +1,684 @@
+    transformed_weights: TransformedWeights,
+    *,
+    output: torch.Tensor | None = None,
+) -> torch.Tensor:
+    if workspace._destroyed:
+        raise RuntimeError("W4A8 workspace has been destroyed")
+    (fc1_weight, fc1_scale), (fc2_weight, fc2_scale) = transformed_weights
+    graph_output = workspace.output if output is None else output
+    weight_key = tuple(
+        tensor.data_ptr()
+        for tensor in (fc1_weight, fc1_scale, fc2_weight, fc2_scale)
+    ) + (graph_output.data_ptr(), workspace._compile_bucket)
```

Ruff (0.16.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)